src/java/org/apache/cassandra/repair/autorepair/AutoRepairState.java

```diff
@@ -101,6 +112,19 @@ protected RepairCoordinator getRepairRunnable(String keyspace, RepairOption opti
                                                  options, keyspace);
     }
 
+    public void calcExpectedScheduleStats(List<PrioritizedRepairPlan> repairPlans)
```

Comment:
nitpick: rename to something like calculateRepairPlanStatistics or updateScheduleStatistics to be more descriptive (feel free to ignore)